crates/gitbutler-tauri/src/worktree/mod.rs

```diff
@@ -76,7 +75,7 @@ pub enum Status {
     },
     Rename {
         #[serde(rename = "previousPath")]
-        previous_path: BString,
+        previous_path: BStringForFrontend,
```

Comment:
Thank you, this was an oversight!

The JSON samples in the tests would have caught it, but I never tested a rename 😁.